Source/ARA/ProcessorEditor.cpp

```diff
@@ -61,7 +61,7 @@ HARPProcessorEditor::HARPProcessorEditor(
   modelPathTextBox.setReadOnly(false);
   modelPathTextBox.setScrollbarsShown(false);
   modelPathTextBox.setCaretVisible(true);
-  modelPathTextBox.setText("path to a gradio endpoint");  // Default text
+  modelPathTextBox.setText("path to a gradio endpoint (see https://github.com/audacitorch/HARP#available-models)");  // Default text
```

Comment:
Yeah, it's just text users can highlight and copy/paste into their browser. I figured it might be better to only display the url initially at the point in the flow where it would be the most useful (i.e. upon opening the HARP plugin), rather than having the link as a permanent fixture.

Reply:
Maybe we could have some sort of tooltip with the link so that it can still be accessed if the default text is deleted.

Reply:
I think you can just lay a `HyperLinkButton` below the process button on the bottom right corner so that people can click on it at all times? That way they can just click and not copy/paste

Reply:
a juce::Label that says "to see a list of available models, see" then the HyperLinkButton with the actual link next to it